src/localization/generalStrings.json

```diff
@@ -50,7 +50,8 @@
     "hours": "hours",
     "minutes": "minutes",
     "no_location_types": "You don't have any location types. Contact your administrator.",
-    "from": "from"
+    "from": "from",
+    "no_locations_for_batch": "There are no locations available for this batch. Contact your administrator."
```

Comment:
I think this message could be different in those cases in which there are location types but are not showing up because there is restriction regarding the location_type? Mainly to give a better feedback to the user about what is actually happening 😀

Reply:
great suggestion - 
a58dd0b 

Hows that?

Reply:
Thanks! It looks much better indeed - at least for me

<img width="1323" alt="Screen Shot 2020-06-24 at 9 45 27 AM" src="https://user-images.githubusercontent.com/32987464/85467637-9176ee80-b5ff-11ea-83bd-f44cc2081ac8.png">